packages/inertia-react/index.d.ts

```diff
@@ -1,20 +1,21 @@
 import * as Inertia from '@inertiajs/inertia'
 import * as React from 'react'
+import { renderToString } from "react-dom/server";
 
-type App<
-  PagePropsBeforeTransform extends Inertia.PagePropsBeforeTransform = Inertia.PagePropsBeforeTransform,
-  PageProps extends Inertia.PageProps = Inertia.PageProps
-> = React.FunctionComponent<{
+export type ReactInstance = React.ReactElement;
+export type ReactComponent = React.ReactNode;
```

Comment:
Why are these aliased and re-exported?

Reply:
@sebastiandedeyne This way, it's easier to work with these / swap and test out the changes in case we're referencing the wrong (sub)type somewhere in the code. It's basically a little shortcut that's relevant as long as the whole thing isn't converted to 100% TS (yet), as apart from the team there isn't really anyone actively helping out with this.